src/checks.ts

```diff
@@ -9,3 +15,9 @@ export function isRevolutCheckoutError(
 ): error is RevolutCheckoutError {
   return error != null && error.name === 'RevolutCheckout'
 }
+
+export function isValidRevolutCheckoutLocale(
+  locale: any
```

Comment:
unknown

Reply:
Yeah, this can be tricky with TS, but `.some` works well!

```ts
const VALID_LOCALES = [
  'en',
  'es',
  'pl'
] as const

type Locale = typeof VALID_LOCALES[number]

function isValidLocale(input: unknown): input is Locale {
  return VALID_LOCALES.some((item) => item === input)
}
```